packages/manager/modules/vrack/src/move-dialog/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "vrack_move_dialog_title": "Déplacement d'un datacenter private cloud",
+  "vrack_move_dialog_intro": "Vous avez selectionné un service de type datacenter private cloud. Ce service ne peut être orphelin et doit être transféré obligatoirement vers un autre vRack.",
+  "vrack_move_dialog_warning_notice": "Cette opération entraîne une interruption de service d'environ 5 minutes.",
+  "vrack_move_dialog_choose_vrack_label": "vRack de destination:",
```

```suggestion
  "vrack_move_dialog_choose_vrack_label": "vRack de destination :",
```
(it's not a normal space, it's an umbreakable space, you should copy paste it if you don't know how to write them with your keyboard 🙂 )

---

This is fixed. I copied the text from your comment. Thanks :)